.github/workflows/ci.yml

```diff
@@ -13,6 +13,21 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  test-windows:
```

Comment:
we should add this to the `needs` section on the `done` job at the end of this file

Reply:
done